src/Eshopworld.Core/IBigBrother.cs

```diff
@@ -24,7 +26,7 @@ public interface IBigBrother
         /// This should be populated by <see cref="System.Runtime.CompilerServices"/>, do not populate this manually.
         /// The line number in the source file at which the method is called.
         /// </param>
-        void Publish<T>(
+        Task Publish<T>(
```

Comment:
Won't it be a problem when a synchronous method is replaced by an asynchronous one? In general it's quite dangerous, but in this case it might work in most cases. Anyway, this requires the major version bump up. Otherwise compilation errors might happen.

Reply:
This package isn't a dependency to any of the applications, so the release here won't break anything. We're bumping `Eshopworld.Telemetry` major version to v3.

The compiler will give you a serious warning (green wiggly) if you're not awaiting an async method.

I can bump this one also, it won't make any difference.

Reply:
Well last time I changed that interface we had the same assumption and it caused problems for 4 different teams/people, including, security, checkout and tahoe. So I would agree with with Artur this requires some action to notify users this is a breaking/dangerous change.

Reply:
@dguerin 
Your comment is based on an outdated version, can you look at the current interface and review that instead?